### DISCUSSION TITLE
Hello, I got O3DE broken when installing dpkg I don't know how.

### DISCUSSION THREAD

**Discussion by ghost** [o3de/o3de#6512] [Help Wanted] [answered]
I was trying to install the O3DE package in Ubuntu 20.04 LTS and I got the package broken when doing dpkg in the terminal.

I'm trying to delete it to reinstall it but I get result about sub-process.

`The following packages will be REMOVED:
  o3de
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 18.2 GB disk space will be freed.
Do you want to continue? [Y/n] Y
(Reading database ... 225617 files and directories currently installed.)
Removing o3de (21.11) ...
dpkg: error processing package o3de (--remove):
 installed o3de package post-removal script subprocess returned error exit status 1
dpkg: too many errors, stopping
Errors were encountered while processing:
 o3de
Processing was halted because there were too many errors.
E: Sub-process /usr/bin/dpkg returned an error code (1)
`
Edit: I also can't install any other package since this seems to be a problem now.
I can't do anything.

Edit 2: Nevermind, the software of the .deb package for Linux isn't corrupted. my package must of got cut of when downloading and I installed it then got corrupted. This error is only fixed when moving the /var/lib/dpkg/info/o3de.(formats) to /tmp and then updating the package manager again & reinstall it again.

Sorry for writing this nonsense but if anyone have gotten a problem installing in the .deb I hope this helps? (dunno)

**Comment by m5able1230** [comment, marked as answer]
sorry for my english 
but i will give you source how to fix your system + install o3de 


how to fix your system  : https://phoenixnap.com/kb/fix-sub-process-usr-bin-dpkg-returned-error-code-1
use method 1 + 3 + 4 + 5 
ignore method 2


how to install o3de : https://o3de.org/docs/welcome-guide/setup/installing-linux/
system requirements : https://o3de.org/docs/welcome-guide/requirements/

you need install cmake + clang 
and install o3de with apt 
i hope it was helpfull 
thankyou
